app/views/posts/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<main>
+  <section class="card-container">
+    <div class="card flex"> 
+      <div class="photo-container"></div>
+      <div class="user-information flex">
+        <h1><%= @user.name %></h1>
+        <p>Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
+  </section>
```

Comment:
- [ ] Dear @dazehinn, It's my belief there was a manual change to the migration file, This caused an issue while I try to migrate the project and start it. As you can see there is a migration pending but could not migrate the project. This prevents me from checking the project locally. Please make sure its possible to drop and migrate your project before the next review request. 
### The error on the console. 
![image](https://github.com/dazehinn/BlogApp/assets/69077061/b6123950-cc03-4201-807a-16a0305ae5b2)
### On the browser
![image](https://github.com/dazehinn/BlogApp/assets/69077061/caaff447-92cc-4933-9491-eaa065121d66)